nginx.tmpl

```diff
@@ -655,7 +655,17 @@ server {
         return 444;
     }
         {{- end }}
-    return 503;
+
+        {{- if (exists "/usr/share/nginx/html/50x.html") }}
+    error_page 500 502 503 504 /50x.html;
+    location /50x.html {
+        root /usr/share/nginx/html;
+        internal;
+    }
+        {{- end }}
+    location ^~ / {
```

Comment:
In case the user navigates to "example.com" or "www.example.com" he gets a 503, right?

If so: how to serve a custom root-Page without the need of a separate nginx serving just that root path?

Reply:
I'm not certain I understand the question. 🤔 

This `server` block (the one which is modified) is a catch all for when you're requesting a hostname that does not exist on a particular instance of nginx-proxy (It's never used when real proxying takes place). The purpose of the PR is to allow customisation of the default error page you get when this happen.

The `location` block is required because having only `return 503` does not trigger the `error_page`.